importfix/shapely/_geometry_helpers.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+
+from shapely__geometry_helpers import *
```

Comment:
It is the * import actually useful, since we import from the same module a line below?

Reply:
Very much so, the `*` imports everything present in the module's `__all__` attribute, while the following line imports things that _aren't_ in `__all__`.
This is necessary so that the fully-namespaced pure python module (`shapely._geometry_helpers`) behaves _exactly_ the same as the built-in compiled module it masquerades as, so no calling code can tell the difference.

Reply:
Gotcha, thanks for clarification!